www/src/data/sidebars/doc-links.yaml

```diff
@@ -222,26 +218,28 @@
               link: /docs/centralizing-your-sites-navigation/
             - title: Rendering sidebar navigation dynamically*
               link: /docs/rendering-sidebar-navigation-dynamically/
+        - title: Making your site accessible
+          link: /docs/making-your-site-accessible
     - title: Improving performance
       link: /docs/performance/
       items:
-        - title: Progressive web app (PWA)
+        - title: Supporting progressive web apps (PWAs)
           link: /docs/progressive-web-app/
         - title: Caching
           link: /docs/caching/
-        - title: Local HTTPS
+        - title: Using local HTTPS
           link: /docs/local-https/
-        - title: Audit with Lighthouse
+        - title: Auditing with lighthouse
           link: /docs/audit-with-lighthouse/
-        - title: Add a Manifest File
+        - title: Addding a manifest file
           link: /docs/add-a-manifest-file/
-        - title: Add Offline Support with a Service Worker
+        - title: Addding offline support with a service worker
```

Comment:
delete one "d"

Reply:
Should the subsequent words after 'Adding' be title cased? (consistent with my comment above)